src/bank_app/bank_app.py

```diff
@@ -0,0 +1,72 @@
+class BankAccount:
+    def __init__(self, balance: float, account_number: str, account_holder: str):
+        self.__balance = balance
+        self.__account_number = account_number
+        self.__account_holder = account_holder
+        self.__transaction_history = []
+        self.__overdraft_limit = 0
+
+    def get_balance(self):
+        return self.__balance
+
+    def add_balance(self, amount):
```

Comment:
Please add docstirngs to class method

Reply:
[Added docstrins in bank_app.py](https://github.com/Lukk1515/bank_app/pull/1/commits/2e50a2b4fe468c2774318e44a563e8d8d809876a)